.github/workflows/markdown-ci.yml

```
@@ -0,0 +1,33 @@
+name: Lint and Check Markdown
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'docs/**'
+      - '**.md'
+      - '**.markdown'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  lint-markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - run: |
+          npm install markdownlint-cli
+      - run: |
+          node_modules/.bin/markdownlint -c .markdownlint.jsonc -i node_modules/ .
```

If you move markdownlint to tools change to 
```suggestion
          node_modules/.bin/markdownlint -c ./tools/.markdownlint.jsonc -i node_modules/ .
```